tests/libstorage/pvc.go

```diff
@@ -163,10 +162,7 @@ func createPVC(pvc *k8sv1.PersistentVolumeClaim, namespace string) *k8sv1.Persis
 }
 
 func CreateFSPVC(name, namespace, size string, labels map[string]string) *k8sv1.PersistentVolumeClaim {
-	sc, exists := GetRWOFileSystemStorageClass()
-	if !exists {
-		Skip("Skip test when RWOFileSystem storage class is not present")
```

Comment:
Yeah, I have yet to run a provisioner that doesn't do RWO FS. If it can do block devices, it can put a filesystem on top 

Anyway - it will fail if the support isn't there

Reply:
just need to be aware that if there are surprised failures it might be because of this, but yeah I also think we dont have lanes with provisioners which doesnt support RWO FS